Assembler/Init.s

```diff
@@ -61,14 +61,30 @@ UPDATE_REFS target @ refs
 .segment "PRG5"   :bank $05 :size $4000 :mem $8000 :off $14010
 .segment "PRG6"   :bank $06 :size $4000 :mem $8000 :off $18010
 .segment "PRG7"   :bank $07 :size $4000 :mem $c000 :off $1c010
-.segment "CHR"    :size $20000 :off $20010 :out
+.segment "PRG10"   :bank $10 :size $2000 :mem $8000 :off $20010
```

Comment:
New banks. Do avoid having to make a bunch of changes, PRG0-PRG7 are left as is as 4K banks.